exchanges/bitfinex/bitfinex_types.go

```diff
@@ -14,6 +15,7 @@ var (
 	errSetCannotBeEmpty = errors.New("set cannot be empty")
 	errSubNotFound      = errors.New("could not find matching subscription")
 	errTypeAssert       = errors.New("type assertion failed")
+	errUnknownError     = errors.New("unknown error")
```

Comment:
Was looking for a common error for this but didn't find it.
Considered making it common.